ToDoListTake2/Program.cs

```diff
@@ -22,7 +22,6 @@ static void Main(string[] args)
         bool running = true;
         int userInput;
         var userAnswer = "";
```

Comment:
They say it is best practice for us to use var instead of being explicit with our types as long as the compiler can figure out what it should be from your initialization of it. So I think it's ok, unless its unclear.